features/step_definitions/user_steps.rb

```diff
@@ -166,6 +166,16 @@
   work.creatorships.unapproved.each(&:accept!)
 end
 
+Given /^"(.*?)" has an empty series "(.*?)"$/ do |login, title|
+  new_series = Series.new
```

Comment:
You don't need to add all pseuds to the series, just one will do. You can write this method shorter like this:
```rb
series = Series.new(title: title)
series.creatorships.build(pseud: User.find_by(login: login).default_pseud)
series.save
```